nixos/modules/services/torrent/qbittorrent-nox.nix

```diff
@@ -0,0 +1,106 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+let
+  cfg = config.services.qbittorrent-nox;
+in
+{
+  meta = {
+    doc = ./qbittorrent-nox.md;
+    maintainers = with lib.maintainers; [ redhawk ];
+  };
+
+  options.services.qbittorrent-nox = {
```

Comment:
IMO the service should be called `qbittorent`.

Reply:
No, qbittorrent is a completely different beast. I do not believe you can run it headless like you can with the webui.

Reply:
I'm not saying you can (though I think it'd be easier than you might think, on Xorg you probably just need to import the `DISPLAY` env variable)

My point is this service is to used to run/host qBittorent. That it is running the `-nox` package is an implementation detail, and should _not_ be surfaced in the service name.

Reply:
> I'm not saying you can (though I think it'd be easier than you might think, on Xorg you probably just need to import the `DISPLAY` env variable)
> 
> My point is this service is to used to run/host qBittorent. That it is running the `-nox` package is an implementation detail, and should _not_ be surfaced in the service name.

Until an actual user wants to do this lets leave it be for now, you're overthinking this. Also isn't xorg entering a deprecation phase? Why would we add more support for a project it's own developers don't even want to work on.

Reply:
Again, I'm not saying we need to figure out how to run the GUI version (that's not relevant). But the naming is an issue, I still think this should be renamed before getting merged.

Reply:
And I still disagree, just because the main qbittorrent package is named that and split into 3 packages does not mean this service should be named after it. If this service only works for qbittorrent nox it should be named after qbittorrent nox. It is HIGHLY relevant here, you are mistaken.